processor/processor.go

```diff
@@ -2796,15 +2797,19 @@ func (proc *Handle) transformSrcDest(
 
 		trace.WithRegion(ctx, "UserTransform", func() {
 			startedAt := time.Now()
-			response = proc.transformer.UserTransform(ctx, eventList, proc.config.userTransformBatchSize.Load())
+			if !proc.config.enableTransformationV2 {
```

Comment:
I suppose we plan to delete this branching sooner rather than later

Reply:
Yes... this is only for backwards compatibility in case something goes bas